src/main/java/com/gooddata/auditevent/AuditEventFilter.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.auditevent;
+
+import static org.apache.commons.lang3.Validate.notNull;
+
+import com.gooddata.collections.PageRequest;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.springframework.beans.BeanUtils;
+import org.springframework.web.util.UriComponentsBuilder;
+
+/**
+ * Class to encapsulate time filtering and paging parameters
+ */
+public class AuditEventFilter extends PageRequest {
```

Comment:
Is "Filter" good naming? (It extends PageRequest which implements Page.)

Reply:
we use this naming in c4 client. your suggestion?

Reply:
"AuditEventPageRequest"?